src/routes/pricing/+page.svelte

```diff
@@ -75,7 +75,7 @@
 										</header>
 										<div class="aw-pricing-cards-content">
 											<ul class="aw-checked-list-circle">
-												<li><span>100GB bandwidth</span></li>
+												<li><span>10GB bandwidth</span></li>
```

Comment:
There's already another PR for this fix: https://github.com/appwrite/website/pull/205. Would you please fix the function execution inconsistency?